### ISSUE TITLE
Latest update 4.51.2 says to run a strings command in terminal to check version of GLIBCXX b4 updating to Node20

### ISSUE THREAD

**Issue by schlotz** [oznu/homebridge-config-ui-x#1619]
### Describe The Bug

I copy pasted the command but got this:

Matt@The-iMac-3 ~ % strings /usr/lib/arm-linux-gnueabihf/libstdc++.so.6 | grep GLIBCXX
sh: line 1: 36371 Bus error: 10           /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -find strings 2> /dev/null
strings: error: unable to find utility "strings", not a developer tool or in PATH

Currently running Homebridge v4.50.6 on an iMac with Node 18.17.1

Bottom line: am I ok to run this update?

### Homebridge UI Version

4.50.6

### Homebridge Version

1.6.1

### Node.js Version

18.17.1

### Operating System

macOS

### Environment Info

- [ ] Using Docker?
- [ ] Using Hyper-V?
- [ ] Using hb-service?

### Raspberry Pi Model

None

**Comment by NorthernMan54** [comment]
I think we need to update the message to indicate that it only applies to RPI platforms.

With a Mac, you need to do the NodeJS outside of the homebridge environment - https://github.com/homebridge/homebridge/wiki/Install-Homebridge-on-macOS
